packages/orchestrator/lib/clients/client.ts

```diff
@@ -344,7 +370,6 @@ export class OrchestratorClient {
         }
     }
 
-    //TODO: rename to cancelTask?
```

Comment:
I don't think we are gonna need a cancelSync.

Reply:
Can you explain what you mean by this exactly?

Reply:
The `cancel` function is cancelling a task (aka: a sync execution). Yesterday I though we might need to have a `cancelSchedule` so I added a TODO to rename the existing function to `cancelTask`. But it happens that we don't cancel schedule (which are only paused/unpaused or deleted), only tasks/jobs.